conf/test_full.config

```diff
@@ -64,6 +64,8 @@ params {
     run_motus                           = true
     motus_save_mgc_read_counts          = true
 
+    run_ganon                           = true
```

Comment:
Shall we activate all ganon arguments here?

Reply:
Good idea!